pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -52,11 +52,13 @@ const (
 	InvitationID = "invitationID"
 )
 
-// message type to store data for eventing. This is retrieved during callback.
-type message struct {
+// Message type to store data for eventing. This is retrieved during callback.
+type Message struct {
```

Comment:
This was originally created for saving transient eventing data. Is it possible to use other structs or update the GoDoc of this struct.?

Reply:
i will update the go doc of this struct